README.md

```diff
@@ -1,23 +1,26 @@
 # puma - Plotting UMami Api
 
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black) 
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Umami docs](https://img.shields.io/badge/info-documentation-informational)](https://umami-hep.github.io/puma/)
 [![PyPI version](https://badge.fury.io/py/puma-hep.svg)](https://badge.fury.io/py/puma-hep)
 
-The Python package `puma` provides a plotting API for commonly used plots in flavour tagging.
+![Testing workflow](https://github.com/umami-hep/puma/actions/workflows/testing.yml/badge.svg)
+![Linting workflow](https://github.com/umami-hep/puma/actions/workflows/linting.yml/badge.svg)
+![Pages workflow](https://github.com/umami-hep/puma/actions/workflows/pages.yml/badge.svg)
 
-ROC curves | Histogram plots | Variable vs efficiency |
-:---------:|:--------------: | :--------------------: |
-<img src=https://umami-docs.web.cern.ch/ci_assets/roc.png width=200> | <img src=https://umami-docs.web.cern.ch/ci_assets/histogram_discriminant.png width=220> | <img src=https://umami-docs.web.cern.ch/ci_assets/pt_light_rej.png width=220> |
+The Python package `puma` provides a plotting API for commonly used plots in flavour tagging.
 
+|                              ROC curves                              |                                     Histogram plots                                     |                            Variable vs efficiency                             |
+| :------------------------------------------------------------------: | :-------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------: |
+| <img src=https://umami-docs.web.cern.ch/ci_assets/roc.png width=200> | <img src=https://umami-docs.web.cern.ch/ci_assets/histogram_discriminant.png width=220> | <img src=https://umami-docs.web.cern.ch/ci_assets/pt_light_rej.png width=220> |
 
 ## Installation
 
 The `puma` package is currently under construction. It can be installed from PyPI or
 using the latest code from this repository.
 
-```bash   
+```bash
 pip install puma-hep
 # or
```

Comment:
we should have a statement here, that via the GitHub link one can get the latest version while on pypi only tagged versions are available